src/components/Progress/ProgressBar/style/ProgressBar.style.ts

```diff
@@ -0,0 +1,129 @@
+import styled, { css, keyframes } from 'styled-components';
+import { Text } from '../../../Typography';
+import { Position, Type } from '../../../../../lib';
+
+interface StyledProgressProps {
+    positionText?: Position;
+}
+
+interface StyledBarProps {
+    positionText?: Position;
+}
+
+interface StyledFillProps {
+    type?: Type;
+}
+
+interface StyledFillContainerProps {
+    progress: Readonly<number>;
+}
+
+interface StyledProgressTextProps {
+    positionText?: Position;
+}
+
+export const StyledProgressBar = styled.div<StyledProgressProps>`
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+
+    ${(props) =>
+        (props.positionText === 'left' || props.positionText === 'right') &&
+        css`
+            align-items: center;
+        `}
+
+    ${(props) =>
+        (props.positionText === 'top' || props.positionText === 'bottom') &&
+        css`
+            flex-direction: column;
+        `}
+`;
+
+export const StyledBar = styled.div<StyledBarProps>`
+    width: 100%;
+    height: 10px;
+    position: relative;
+    background: ${(props) => props.theme.color.background.gamma};
+    border-radius: ${(props) => props.theme.radius};
+
+    ${(props) =>
+        (props.positionText === 'right' || props.positionText === 'bottom') &&
+        css`
+            order: 1;
+        `}
+    ${(props) =>
+        (props.positionText === 'left' || props.positionText === 'right') &&
+        css`
+            margin: 0 ${(props) => props.theme.margin.sm};
+        `};
+`;
+
+export const StyledFillContainer = styled.div<StyledFillContainerProps>`
+    width: ${(props) => props.progress}%;
+    height: 100%;
+    transition: width 2s ease;
+`;
+
+const animationFill = keyframes`
+        0% {
+            width: 0%;
+        }
+
+        100% {
+            width: 100%;
+        }
+`;
+
+export const StyledFill = styled.span<StyledFillProps>`
+    display: block;
+    width: 100%;
+	height: 100%;
+	position: relative;
+	overflow: hidden;
+	box-sizing: border-box;
+	margin: 0;
+	border-radius: ${(props) => props.theme.radius};
```

Comment:
U tohoto řádku a výš nesedí odsazení

Reply:
tady právě sedí, ale u keyframes nesedělo, takže to jsem zarovnala a je to vše stejne

Reply:
Stále máš u `width` a `display` jiné formátování než u `height` a níž.